static/src/javascripts/projects/common/modules/experiments/ab-tests.js

```diff
@@ -1,6 +1,7 @@
 // @flow
 import { commercialPrebidSafeframe } from 'common/modules/experiments/tests/commercial-prebid-safeframe.js';
 import { commercialAdVerification } from 'common/modules/experiments/tests/commercial-ad-verification.js';
```

Comment:
Is the `commercialAdVerification` test being used anywhere now? It looks like `commercialAdVerificationQA` will replace it in `prepare-ad-verification.js` (the only place it's used). Can we remove this test?

Reply:
Not currently being used but planned to be used in the next phase, hence left untouched.